presto-docs/src/main/sphinx/release/release-0.200.rst

```diff
@@ -0,0 +1,31 @@
+=============
+Release 0.200
+=============
+
+General Changes
+---------------
+
+* Disable early termination of inner/right joins when the right side has zero rows.
+  This optimization can cause indefinite query hangs for eligible queries.
```

Comment:
This might be misleading. A query can be affected if the right side can be affected if row count is small enough or skewed enough. I believe queries with zero rows is actually not affected.

Suggestions?

Reply:
How about "for eligible queries" -> "for queries that join against a small number of rows"